API/schema/opportunities-schema.txt

```diff
@@ -1,34 +1,81 @@
+//Database
 {
     //Here stores all the opportunities
+    "opportunities":[
+        // One opportunity
+        {
+            "id":int,
+            "category":str,
```

Comment:
Should this be from a determined set of categories? (From UI perspective, this should be dropdown menu rather than text input). Will make filtering on category make more sense